### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or a supporter signed this repo up for Open Collective. This pull request adds backers and sponsors from your Open Collective https://opencollective.com/sweetalert2 ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

**Comment by monkeywithacupcake** [comment]
I was not sure if you wanted to incorporate the Donors on Open Collective and other platforms, so I just kept them separate.

**Comment by zenflow** [comment]
<3 this.. Thank you @monkeywithacupcake!!!

How do the rest of the collaborators feel about this?

I will do a real review soon, provided that the others like it too.

File: package.json [hunk 1]
```diff
@@ -85,8 +85,17 @@
...
-    "release": "node release"
+    "release": "node release",
+    "postinstall": "opencollective postinstall"
   },
   "bugs": "https://github.com/sweetalert2/sweetalert2/issues",
-  "license": "MIT"
-}
+  "license": "MIT",
+  "dependencies": {
+    "opencollective": "^1.0.3"
```
Comment (limonte) [inline package.json]: this should be in devDeps I believe

File: package.json [hunk 2]
```diff
@@ -85,8 +85,17 @@
...
+  },
+  "collective": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/SweetAlert2",
+    "logo": "https://opencollective.com/SweetAlert2/logo.txt"
```
Comment (limonte) [inline package.json]: <3

File: README.md [hunk 3]
```diff
@@ -192,3 +195,33 @@ Has SweetAlert2 helped you create an amazing application? You can show your supp
 - https://www.patreon.com/limonte
 
 ### [Hall of Donators :trophy:](DONATIONS.md)
+
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```
Comment (zenflow) [inline README.md]: We have no `CONTRIBUTING.md` but we have a Contributing section in this README.md. This could link to `#contributing`

**Comment by zenflow** [comment]
We could probably merge this and just make the minor fixes ourselves.. @monkeywithacupcake has already helped quite a lot, and probably has more of these PRs to make. 

Thank you so much @monkeywithacupcake !!!

**Comment by limonte** [comment]
> We could probably merge this and just make the minor fixes ourselves.. @monkeywithacupcake has already helped quite a lot, and probably has more of these PRs to make.
>
> Thank you so much @monkeywithacupcake !!!

Good idea, @zenflow could you handle those small details?

**Comment by zenflow** [comment]
@limonte You got it

File: package.json (hunk 1 above)
Comment (monkeywithacupcake) [reply package.json]: Actually this is a dependency and not a dev dependency. It makes a network request. Two alternatives:

One user made a lighter version of the post install: https://github.com/WebReflection/lightercollective
You can also remove the dependency altogether: https://github.com/remy/nodemon/blob/master/package.json#L42

**Pull request merged**